requirements/pip.txt

```diff
@@ -8,7 +8,7 @@ wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==22.2.2
+pip==22.3
```

Comment:
Some deprecations in the new version. `--no-cache` deprecation may affect some builds. For details, See https://pip.pypa.io/en/stable/news/#v22-3